.golangci.yml

```diff
@@ -102,7 +102,7 @@ linters-settings:
     #local-prefixes: github.com/org/project
   gocyclo:
     # minimal code complexity to report, 30 by default (but we recommend 10-20)
-    min-complexity: 12
+    min-complexity: 13
```

Comment:
I'm unhappy about this, but willing to do it because it's just a bump of 1. No more than that please. The refactor I did in https://github.com/matrix-org/dendrite/pull/917/commits/5f7ff41ed96dd460e3d6ef8278d8e8723adab6c3 because of gocyclo I think feels a lot better, and I wouldn't have done it with this at 13.

Reply:
I've increased this by 1 because there are a few places now where adding *anything*, even simple things, takes us over the cyclo limit due to other changes, like in SQLite where the `common.WithTransaction` wraps automatically take a cyclo away from us, or in cases where we now care about room versions and previously didn't. There isn't otherwise a need to refactor those things and to do so would make some code paths look *less* clear.